examples/campaign_management/create_campaign_experiment.rb

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example illustrates how to begin creation of a campaign experiment from
+# a draft and wait for it to complete.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def create_campaign_experiment(customer_id, campaign_draft_resource_name)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  experiment = client.resource.campaign_experiment do |ce|
+    ce.campaign_draft = client.wrapper.string(campaign_draft_resource_name)
+    ce.name = client.wrapper.string(
+      "Campaign Experiment ##{(Time.new.to_f * 1000).to_i}"
+    )
+    ce.traffic_split_percent = client.wrapper.int64(50)
+    ce.traffic_split_type = :RANDOM_QUERY
+  end
+
+  # A Google::Longrunning::Operation (LRO) is returned from this asynchronous
+  # request by the API, and is converted into a Google::Gax::Operation
+  # automatically.
+  operation = client.service.campaign_experiment.create_campaign_experiment(
+    customer_id,
+    experiment
+  )
+
+  puts "Created campaign experiment with resource name " +
+      "#{operation.metadata.campaign_experiment}"
+
+  puts "Waiting until operation completes."
+
+  # The wait_until_done! method implements a default backoff policy for
+  # retrying.
+  operation.wait_until_done! do |op|
+    raise op.results.message if operation.error?
+  end
+
+  # You can also use operation.refresh! to make a call to the API to check
+  # whether the LRO is finished, and operation.done? after refreshing to check
+  # the status, if you'd rather implement your own backoff logic.
+
+  query = <<~EOQUERY
```

Comment:
Worth mentioning here that upon completion of the LRO, the operation's metadata will be a StringValue containing the resource name of the created experiment, and that the query is to get the resource name of the corresponding experiment campaign.